### ISSUE TITLE
点击一键训练，不知道结束训练了没有，日志卡住没有输出了

### ISSUE THREAD

**Issue by hjj-lmx** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#2532]
2025-04-01 18:53:10 | INFO | __main__ | Use gpus: 0-1
2025-04-01 18:53:10 | INFO | __main__ | Execute: "D:\Program Files\miniconda3\envs\t2a\python.exe" infer/modules/train/train.py -e "mi-test" -sr 40k -f0 1 -bs 12 -g 0-1 -te 20 -se 5 -pg assets/pretrained_v2/f0G40k.pth -pd assets/pretrained_v2/f0D40k.pth -l 0 -c 0 -sw 0 -v v2
[W socket.cpp:663] [c10d] The client socket has failed to connect to [DESKTOP-L2OV9PU]:52550 (system error: 10013 - ��һ�ַ���Ȩ�޲�����ķ�ʽ����һ�������׽��ֵĳ��ԡ�).
[W socket.cpp:663] [c10d] The client socket has failed to connect to [kubernetes.docker.internal]:52550 (system error: 10049 - �����������У�������ĵ�ַ��Ч��).
INFO:mi-test:{'data': {'filter_length': 2048, 'hop_length': 400, 'max_wav_value': 32768.0, 'mel_fmax': None, 'mel_fmin': 0.0, 'n_mel_channels': 125, 'sampling_rate': 40000, 'win_length': 2048, 'training_files': './logs\\mi-test/filelist.txt'}, 'model': {'filter_channels': 768, 'gin_channels': 256, 'hidden_channels': 192, 'inter_channels': 192, 'kernel_size': 3, 'n_heads': 2, 'n_layers': 6, 'p_dropout': 0, 'resblock': '1', 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'resblock_kernel_sizes': [3, 7, 11], 'spk_embed_dim': 109, 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4], 'upsample_rates': [10, 10, 2, 2], 'use_spectral_norm': False}, 'train': {'batch_size': 12, 'betas': [0.8, 0.99], 'c_kl': 1.0, 'c_mel': 45, 'epochs': 20000, 'eps': 1e-09, 'fp16_run': True, 'init_lr_ratio': 1, 'learning_rate': 0.0001, 'log_interval': 200, 'lr_decay': 0.999875, 'seed': 1234, 'segment_size': 12800, 'warmup_epochs': 0}, 'model_dir': './logs\\mi-test', 'experiment_dir': './logs\\mi-test', 'save_every_epoch': 5, 'name': 'mi-test', 'total_epoch': 20, 'pretrainG': 'assets/pretrained_v2/f0G40k.pth', 'pretrainD': 'assets/pretrained_v2/f0D40k.pth', 'version': 'v2', 'gpus': '0-1', 'sample_rate': '40k', 'if_f0': 1, 'if_latest': 0, 'save_every_weights': '0', 'if_cache_data_in_gpu': 0}
[W socket.cpp:663] [c10d] The client socket has failed to connect to [DESKTOP-L2OV9PU]:52550 (system error: 10013 - ��һ�ַ���Ȩ�޲�����ķ�ʽ����һ�������׽��ֵĳ��ԡ�).
[W socket.cpp:663] [c10d] The client socket has failed to connect to [kubernetes.docker.internal]:52550 (system error: 10049 - �����������У�������ĵ�ַ��Ч��).

“特征检索库文件路径,为空则使用下拉的选择结果” ，这个值应该怎传
